### DISCUSSION TITLE
Is this package only support PHP 8

### DISCUSSION THREAD

**Discussion by tawiparkP** [spatie/laravel-site-search#17] [Q&A] [answered]
Hello, I've tried to install the package via composer on my php 7.3 project but I got 

"Package spatie/laravel-site-search has a PHP  requirement incompatible with your PHP version, PHP extensions and Composer version"

Do I need to build php 8 project to use the package or there're some php 7 supported somewhere?

**Comment by freekmurze** [comment, marked as answer]
PHP 7 isn't supported.
